README.md

```diff
@@ -1,2 +1,10 @@
 # benchmark
 Tools to test scalability &amp; performance of Dispatch
+
+# Requirements
```

Comment:
This should say `go get dep &&  dep ensure`. Or,  ideally, just vendor the dependencies.